scripts/train/train.py

```diff
@@ -482,6 +480,8 @@ def main(cfg: DictConfig) -> Trainer:
         device_train_batch_size,
     )
 
+    use_async_eval = any(isinstance(c, AsyncEval) for c in callbacks)
+
```

Comment:
why is this here?

Reply:
should be moved back where it was, although doesn't really matter

Reply:
moved it back